features/record/components/organisms/distance-page-modal.tsx

```diff
@@ -90,9 +90,14 @@ export function DistancePageModal({
   };
 
   const handleDoneButtonClick = () => {
-    if (secondaryTabIndex === 0 && assistiveTabIndex === 0)
-      setValue('totalDistance', totalMeter + 'm');
-    else setValue('totalDistance', totalStrokeDistance + 'm');
+    if (secondaryTabIndex === 0 && isAssistiveIndexZero) {
+      if (totalMeter) setValue('totalDistance', totalMeter + 'm');
+      else setValue('totalDistance', undefined);
+    } else {
+      if (totalStrokeDistance)
+        setValue('totalDistance', totalStrokeDistance + 'm');
+      else setValue('totalDistance', undefined);
+    }
```

Comment:
```suggestion
    if (secondaryTabIndex === 0 && isAssistiveIndexZero) {
      setValue('totalDistance', totalMeter ? totalMeter + 'm' : undefined);
    } else {
      setValue('totalDistance', totalStrokeDistance ? totalStrokeDistance + 'm' : undefined);
    }
```
setValue 안에서 삼항 연산자로 나누어 처리해도 깔끔할것 같다는 생각이 듭니다!

Reply:
혹은

```js
if (secondaryTabIndex === 0 && isAssistiveIndexZero) {
  setValue('totalDistance', totalMeter && totalMeter + 'm');
} else {
  setValue('totalDistance', totalStrokeDistance && totalStrokeDistance + 'm');
}
```
이렇게?